meson.build

```diff
@@ -2500,6 +2500,15 @@ if conf.get('ENABLE_OOMD') == 1
                    install_rpath : rootlibexecdir,
                    install : true,
                    install_dir : rootlibexecdir)
+
+        executable('oomctl',
+                   oomctl_sources,
+                   include_directories : includes,
+                   link_with : [libshared],
+                   dependencies : [],
+                   install_rpath : rootlibexecdir,
+                   install : true,
```

Comment:
We should probably remove the conditional and instead say `install : conf.get('ENABLE_OOMD') == 1`. But that should be changed for all executable, so it's appropriate for this PR to match the surrounding code.